.github/actions/winget-publish/action.yaml

```
@@ -0,0 +1,40 @@
+name: Winget Publish
+description: A composite action to publish packages to the Windows Package Manager (Winget) repository
+
+inputs:
+  release-version:
+    description: "Version to publish to Winget package manager (without 'v' prefix)"
+    required: true
+  package-id:
+    description: "Package identifier in Winget repository"
+    required: true
+    default: "Deskflow.Deskflow(64-bit)"
```

```suggestion
    default: "Deskflow.Deskflow"
```

I wonder if 64-bit is redundant, since we don't offer a 32-bit MSI.